src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -173,7 +173,7 @@ public static void deleteColony(final int id)
                     citizenData.getCitizenEntity().onDeath(CONSOLE_DAMAGE_SOURCE);
                     colonyWorlds.add(world);
                 }
-            }
+            }if(canDestroy){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS00122)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)